src/components/ContentOpenWith/OpenWithButton.js

```diff
@@ -72,7 +70,7 @@ const OpenWithButton = ({
         >
             <Button
                 isDisabled={isDisabled}
-                onClick={onClick ? () => onClick(id) : noop}
+                onClick={onClick(displayIntegration)}
```

Comment:
This changed from a function declaration to a direct invocation. Was that intentional?

Reply:
Fixed